lib/rubocop/rspec/description_extractor.rb

```diff
@@ -54,7 +55,7 @@ def class_documentation?
         end
 
         def rspec_cop_namespace?
-          documented_constant.start_with?(COP_NAMESPACE)
+          documented_constant.start_with?(RSPEC_NAMESPACE, FACTORIES_NAMESPACE)
```

Comment:
Is `FACTORIES_NAMESPACE` dead code now that it lives inside of `RuboCop::Cop::RSpec`?